test/utils.ts

```diff
@@ -82,6 +79,6 @@ export function rmDir (folder: string): void {
       fs.unlinkSync(path.join(folder, file))
     }
 
-    fs.rmdirSync(config.get<DbBackUpConfig>('dbBackUp').path, { recursive: true })
+    fs.rmdirSync(folder, { recursive: true })
```

Comment:
Good catch, although don't understand why you have the above `if` with `for`? Shouldn't the `{recursive: true}` handle this for you?

Reply:
`{recursive: true}` can not remove folder if folder contains some files